src/utils/patterns.ts

```diff
@@ -210,24 +202,25 @@ const selectApplicationPineOptions = {
 			$select: 'slug',
 		},
 	},
-} satisfies PineOptions<Application>;
+} as const;
 
-type SelectApplicationResult = PineTypedResult<
-	Application,
-	typeof selectApplicationPineOptions
+type SelectApplicationResult = Pine.OptionsToResponse<
+	Application['Read'],
+	typeof selectApplicationPineOptions,
+	undefined
 >;
```

Comment:
NIT: Since `selectApplication` returns 1 application, could we make this type be the object instead of an array?
```suggestion
type SelectApplicationResult = Pine.OptionsToResponse<
	Application['Read'],
	typeof selectApplicationPineOptions,
	undefined
>[number];
```